### PR TITLE
feat: Implementing the list view screen for Admin > organization section

### PR DESCRIPTION
**Pull request by kerlynNkep**
## Description
Added the organization component

## Related issues and discussion
#323 

## Screenshots, if any
organization

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

File: src/app/admin/organization/organization-routing.module.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/** Angular Imports */
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+
+/** Routing Imports */
+import { Route } from '../../core/route/route.service';
+
+/** Translation Imports */
+import { extract } from '../../core/i18n/i18n.service';
+import { OrganizationComponent } from './organization.component';
```
Comment (abhaychawla) [inline src/app/admin/organization/organization-routing.module.ts]: Incorrectly placed above the comment.

File: src/app/admin/organization/organization-routing.module.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const routes: Routes = [
+  Route.withShell([
+    {
+      path: 'organization',
+      data: { title: extract('Organization'), breadcrumb: 'Organization' },
+      children: [
+        {
+          path: '',
+          component: OrganizationComponent
+        }]
```
Comment (abhaychawla) [inline src/app/admin/organization/organization-routing.module.ts]: Square bracket should be closed on the next line.

File: src/app/admin/organization/organization.component.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+  <mat-card>
+
+    <div fxLayout="row" fxLayout.lt-md="column">
+
+      <div fxFlex="50%">
+
+        <mat-nav-list>
+
+          <mat-list-item [routerLink]="[]">
```
Comment (abhaychawla) [inline src/app/admin/organization/organization.component.html]: Remove blank `router links`.

File: src/app/admin/organization/organization.component.html [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+            <mat-icon matListIcon>
+              <i class="fa fa-building"></i>
```
Comment (abhaychawla) [inline src/app/admin/organization/organization.component.html]: Please use font awesome angular icons. Example: `<fa-icon icon="sync" size="sm"></fa-icon>`

File: src/app/admin/organization/organization.component.html [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+            </mat-icon>
+            <h4 matLine>Manage Offices</h4>
+            <p matLine>Add new office or modify or deactive office or modify hierarchy</p>
```
Comment (abhaychawla) [inline src/app/admin/organization/organization.component.html]: Spelling mistake => deactivate

File: src/app/admin/organization/organization.component.html [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+            <h4 matLine>Teller / Cashier Management</h4>
+            <p matLine>Manage tellers / cashier and cash allocation  and settlement</p>
+          </mat-list-item>
+
+          <mat-list-item [routerLink]="[]">
+            <mat-icon matListIcon>
+              <i class="fa fa-calendar"></i>
+            </mat-icon>
+            <h4 matLine>Working Days</h4>
+            <p matLine>Define working days and configure behaviour of payments due to holidays </p>
```
Comment (abhaychawla) [inline src/app/admin/organization/organization.component.html]:
```suggestion
            <p matLine>Define working days and configure behaviour of payments due to holidays</p>
```

File: src/app/admin/organization/organization.module.ts [hunk 7]
```diff
@@ -0,0 +1,20 @@
+/** Angular Imports */
+import { NgModule } from '@angular/core';
+
+/** Custom Modules */
+
+
+/** Custom Components */
+import { OrganizationComponent } from './organization.component';
+import { OrganizationRoutingModule } from './organization-routing.module';
```
Comment (abhaychawla) [inline src/app/admin/organization/organization.module.ts]: Incorrectly placed below wrong comment.

File: src/app/admin/organization/organization.module.ts [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+import { SharedModule } from '../../shared/shared.module';
```
Comment (abhaychawla) [inline src/app/admin/organization/organization.module.ts]: Incorrectly placed below wrong comment.

File: src/app/admin/organization/organization.module.ts [hunk 9]
```diff
@@ -0,0 +1,20 @@
+/** Angular Imports */
+import { NgModule } from '@angular/core';
+
+/** Custom Modules */
+
+
+/** Custom Components */
+import { OrganizationComponent } from './organization.component';
```
Comment (abhaychawla) [inline src/app/admin/organization/organization.module.ts]: Not declared.

File: src/app/app.module.ts [hunk 10]
```diff
@@ -29,9 +29,12 @@ import { CentersModule } from './centers/centers.module';
 import { AccountingModule } from './accounting/accounting.module';
 import { SelfServiceModule } from './self-service/self-service.module';
 import { SystemModule } from './system/system.module';
+import { OrganizationModule } from './admin/organization/organization.module';
 
 /** Main Routing Module */
 import { AppRoutingModule } from './app-routing.module';
+import { OrganizationComponent } from './admin/organization/organization.component';
```
Comment (abhaychawla) [inline src/app/app.module.ts]: Not supposed to be imported here.

File: src/app/app.module.ts [hunk 11]
```diff
@@ -57,9 +60,10 @@ import { AppRoutingModule } from './app-routing.module';
     AccountingModule,
     SelfServiceModule,
     SystemModule,
+    OrganizationModule,
     AppRoutingModule
   ],
-  declarations: [WebAppComponent],
+  declarations: [WebAppComponent, OrganizationComponent],
```
Comment (abhaychawla) [inline src/app/app.module.ts]: Why have you declared this in the main app module? What is the use of organization module?

**Comment by abhaychawla** [comment]
@kerlynNkep It has been a long time since you have been inactive. Shall I expect an update on this PR or close it so any other contributor might work on it?

**Comment by kerlynNkep** [comment]
@abhaychawla am sorry my machine crashed and later got bad. I have also been busy with my graduation. Am so sorry for the delay . Am in process of getting another PC. I will make the changes and push immediately am settled and that will be before the year runs out.

**Comment by kerlynNkep** [comment]
Hello @abhaychawla I have sent a PR. Please can you review it  and get back to me. Please I am also having problems squashing it.

File: src/app/organization/organization-routing.module.ts [hunk 12]
```diff
@@ -0,0 +1,31 @@
+/** Angular Imports */
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import { Route } from '../core/route/route.service';
```
Comment (abhaychawla) [inline src/app/organization/organization-routing.module.ts]: Add all missing comments.

File: src/app/organization/organization.component.html [hunk 13]
```diff
@@ -0,0 +1,164 @@
...
+
+        </mat-nav-list>
+
+      </div>
+
+    </div>
+
+  </mat-card>
+
+</div>
```
Comment (abhaychawla) [inline src/app/organization/organization.component.html]: Add a line at the end of file.

File: src/app/organization/organization.component.ts [hunk 14]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (abhaychawla) [inline src/app/organization/organization.component.ts]: Add all necessary comments.

File: src/app/organization/organization.module.ts [hunk 15]
```diff
@@ -0,0 +1,17 @@
+/** Angular Imports */
+import { NgModule } from '@angular/core';
+import { OrganizationComponent } from './organization.component';
```
Comment (abhaychawla) [inline src/app/organization/organization.module.ts]: Add all necessary comments.

**Comment by abhaychawla** [comment]
@kerlynNkep Are all the requested changes done for this PR?

**Comment by kerlynNkep** [comment]
@abhaychawla Yes I have made the requested changes for this PR

**Comment by abhaychawla** [comment]
@kerlynNkep A few more comments are left to be added. Take reference from `accounting` section and add them. Also, resolve all the conversations when you are done.

**Comment by abhaychawla** [comment]
Closing, as there has been no update for a long time.

**Pull request closed without merging**